src/main.rs

```diff
@@ -594,20 +598,61 @@ const APP: () = {
             let adc1_in_p = gpiof.pf11.into_analog();
             let adc2_in_p = gpiof.pf14.into_analog();
 
-            Some(
-                pounder::PounderDevices::new(
-                    io_expander,
-                    ad9959,
-                    spi,
-                    adc1,
-                    adc2,
-                    adc1_in_p,
-                    adc2_in_p,
-                )
-                .unwrap(),
+            let pounder_devices = pounder::PounderDevices::new(
+                io_expander,
+                &mut ad9959,
+                spi,
+                adc1,
+                adc2,
+                adc1_in_p,
+                adc2_in_p,
             )
+            .unwrap();
+
+            let dds_output = {
+                let io_update_trigger = {
+                    let _io_update = gpiog
+                        .pg7
+                        .into_alternate_af2()
+                        .set_speed(hal::gpio::Speed::VeryHigh);
+
+                    // Configure the IO_Update signal for the DDS.
+                    let mut hrtimer = hrtimer::HighResTimerE::new(
+                        dp.HRTIM_TIME,
+                        dp.HRTIM_MASTER,
+                        dp.HRTIM_COMMON,
+                        ccdr.clocks,
+                        ccdr.peripheral.HRTIM,
+                    );
+
+                    // IO_Update should be latched for 50ns after the QSPI profile write. Profile writes
+                    // are always 16 bytes, with 2 cycles required per byte, coming out to a total of 32
```

Comment:
This is now configurable, but the _limit_ is 16 bytes. This is only limited by the QSPI FIFO size and the size used in the `ad9959` serializer, so it could theoretically change in the future and invalidate these numbers